app/assets/javascripts/modules/organisation-list-filter.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable no-var */
+
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function FilterOrganisations ($module) {
+    this.$module = $module
+    this.filterTimeout = null
+    this.form = this.$module.querySelector('[data-filter="form"]')
+    this.orgResults = this.$module.querySelector('#organisations_search_results')
+  }
+
+  FilterOrganisations.prototype.init = function () {
+    this.$module.filterList = this.filterList.bind(this)
```

Comment:
This may simply be my lack of knowhow around binding in js so sorry if I'm regurgitating something you've already run into: Why do you need to create a separate call for this function? It's only being used on line 31 so could you not just use something like...

```
this.filterList(searchTerm).bind(this)
```

...on line 31 in place of this variable call? There's already loads of binding going on in that timeout and the event listener itself so the bind might not be needed.

Reply:
How bizarre 🤔  There might be something going on with the function being bound before it can be called. If it works it works, I'm not super precious about this. Feel free to resolve.